EmailReporting.php

```diff
@@ -637,6 +650,24 @@ function ERP_update_check( )
 
 			plugin_config_set( 'config_version', 18 );
 		}
+
+		if ( $t_config_version <= 18 )
+		{
+			$t_mailboxes = plugin_config_get( 'mailboxes', array() );
+			$t_indexes = array(
+				'custom_emails' => 'custom_emails',
+				'custom_emails_addresses' => 'custom_emails_addresses',
+			);
+
+			foreach ( $t_mailboxes AS $t_key => $t_array )
+			{
+				$t_mailboxes[ $t_key ] = $this->ERP_update_indexes( $t_array, $t_indexes );
+			}
+
+			plugin_config_set( 'mailboxes', $t_mailboxes );
+
+			plugin_config_set( 'config_version', 19 );
+		}
```

Comment:
ERP_updates_indexes is meant for renaming existing index keys. These are new keys so this should not be necessary.

Reply:
I noticed why you did it this way.
You should add the two indexes to this function: https://github.com/mantisbt-plugins/EmailReporting/blob/master/core/config_api.php#L8-L19